### ISSUE TITLE
res headers is unformatted

### ISSUE THREAD

**Issue by brainsiq** [pinojs/pino-http#32]
Is it possible to format the response headers in the same way request headers appear in the log output?

Currently they appear as:

```
res: {
      "statusCode": 200,
      "header": "Content-Type: application/json; charset=utf-8\r\nContent-Length: 34"
}
```

Ideally they should be:

```
res: {
      "statusCode": 200,
      "headers:" {
           "content-type": "application/json; charset=utf-8\r\n",
           "content-length": 34
      }
}
```

**Comment by mcollina** [comment]
Yes it is. Unfortunately headers are not stored as object in the `Response`, the best we can do is: https://github.com/pinojs/pino/blob/master/lib/serializers.js#L16. I would restrain from parsing them, as it is expensive.

Since v7.7.0 of Node.js there is `res.getHeaders()`, https://nodejs.org/api/http.html#http_response_getheaders. That replaces `_headers` https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js#L128, which is available everywhere. This change is semver-major for pino, as the http res serializer lives there.
